pkg/serverless/protocol.go

```diff
@@ -207,10 +218,12 @@ func StartDaemon(stopTraceAgent context.CancelFunc) *Daemon {
 		mux:              mux,
 		ReadyWg:          &sync.WaitGroup{},
 		InvcWg:           &sync.WaitGroup{},
+		extraTagsSetWg:   &sync.WaitGroup{},
```

Comment:
Would it make sense to  reuse the ReadyWg instead? The /hello route blocks until the ReadyWg completes, so it could be useful to make sure the handler code doesn't start until we are ready.

Reply:
Totally ! 💯